bash/dev_compile_python_packages.sh

```diff
@@ -5,6 +5,7 @@ set -e
 RELATIVE_SCRIPTPATH=$0
 
 # Update and install packages used to compile requirements
+apt-get -yq install python3-venv
```

Comment:
Why is this needed?

Reply:
this became necessary after I changed the dev container Dockerfile to use the `gcr.io/google.com/cloudsdktool/google-cloud-cli` image. after some messing around, this seemed the easiest/most reliable way to get the google cloud SDK in the dev container (it's [used in GHAs](https://github.com/NYCPlanning/db-zap-opendata/blob/build-map-zap/.github/workflows/export.yml#L57-L62))

I didn't mess around too much with optimizing the use of that new image layer and our usual `mcr.microsoft.com/devcontainers/python` image layer, but it only worked when the google one was last, and we _may_ not need the old one at all

note: I ended up not _needing_ the google cloud SDK, but since it's used in other parts of the repo in CI and would be needed for any auto exports/imports from bigquery to file storage, it seems nice to have ready to use locally

Reply:
Gotcha, that makes sense